ansible/configs/satellite-vm/software.yml

```diff
@@ -8,12 +8,15 @@
         msg: "Software tasks Started"
     
 
-- name: Configuring Bastion Hosts
+- name: Configuring satellite Hosts
   hosts: satellites
   become: true
   roles:
-  #  - { role: "rhn-subscription-manager", when: rhn_subscription_manager }
-    - { role: "satellite-installation", when: install_satellite }   
+    # Satellite-installation role installs satellite and Configures firewalld
+    - { role: "satellite-installation", when: satellite_installation }   
```

Comment:
in other configs the variables to enable / disable features usually start with `install_`, so maybe you want to have the same here. `install_satellite` and `configure_satellite`.

Reply:
variable name replaced to install_satellite and configure_satellite